### PR TITLE
Added animations

### LINKED ISSUE #167: Adding animated social media icons in footer and animated illustrations instead of static images
I can add animated social media icons in footer section also animated illustrations, It really looks attractive

social icons used to promote our work on social networks and get in return potential customers. So , I thought there should be something to make it attractive
You can assign this issue under gssoc label

### PR DESCRIPTION
**Pull request by sonalisonawane**
# Description

The static Images of About us and banner sections are changed with animations illustrations images .

Fixed #167

## Type of change

Please check options that are relevant to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have pulled the latest changes in the repository
- [x] I have squashed my commits
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

I am attaching the images that I had added in this page, Actually if in case they looked static here but they are animated, you can go through this repo.:)
AboutUs

banner

### CODE REVIEW

**Comment by vaishali614** [comment]
@sonalisonawane These animated images look good. However, only About Us image seems relevant but not the Banner image. Please try some other illustration.

**Review by abdus — APPROVED** [review]
LGTM

**Comment by DeepanshiD** [comment]
Hey @sonalisonawane this looks great, but don't stretch the first image just keep width as it is to 100%. Second one is perfect.

**Comment by vaishali614** [comment]
@sonalisonawane Please make the changes suggested by @DeepanshiD. Also, when you update your PR, do take the latest pull before doing so.

**Comment by sonalisonawane** [comment]
> @sonalisonawane Please make the changes suggested by @DeepanshiD. Also, when you update your PR, do take the latest pull before doing so.

Do I have to add another one in Banner section ??

**Comment by sonalisonawane** [comment]
> @sonalisonawane Please make the changes suggested by @DeepanshiD. Also, when you update your PR, do take the latest pull before doing so.

@vaishali614 , @DeepanshiD  mam ,
error: RPC failed; curl 18 transfer closed with outstanding read data remaining
fatal: the remote end hung up unexpectedly
fatal: the remote end hung up unexpectedly

this error is occurring while, pushing code.

**Comment by sonalisonawane** [comment]
Done! I had change the banner image also fixed its size. Can you please check it once. @DeepanshiD  Mam , @vaishali614  Mam, @abdus sir

**Comment by sonalisonawane** [comment]
This one is new image :)
banner

File: src/CSS/Banner.module.css [hunk 1]
```diff
@@ -57,7 +57,7 @@
 
 .BannerImage {
   z-index: 10;
-  width: 700px;
+  width: 100%;
   height: 400px;
```
Comment (vaishali614) [inline src/CSS/Banner.module.css]: Increase height to 525px
Add margin-top: -15%

And remove bottom padding from jumbotron to cut extra space above 'About Us' section.

Rest everything looks good.

**Review by vaishali614 — CHANGES_REQUESTED** [review]

**Comment by Manvityagi** [comment]
Awesome work @sonalisonawane

**Pull request merged**